Examples/Run/Common/src/CommonOptions.cpp

```diff
@@ -117,10 +126,36 @@ void FW::Options::addInputOptions(
 
 boost::program_options::variables_map FW::Options::parse(
     const boost::program_options::options_description& opt, int argc,
-    char* argv[]) {
+    char* argv[]) noexcept(false) {
   variables_map vm;
-  store(parse_command_line(argc, argv, opt), vm);
+  store(command_line_parser(argc, argv)
+            .options(opt)
+            .extra_parser(rf_option_parser)
+            .run(),
+        vm);
   notify(vm);
+
+  if (vm.count("response-file") and
+      not vm["response-file"].template as<std::string>().empty()) {
+    // Load the file and tokenize it
+    std::ifstream ifs(vm["response-file"].as<std::string>().c_str());
+    if (!ifs) {
+      throw(std::filesystem::filesystem_error("Could not open response file.",
+                                              std::error_code()));
+    }
+    // Read the whole file into a string
+    std::stringstream ss;
+    ss << ifs.rdbuf();
+    // Split the file content
+    boost::char_separator<char> sep(" \n\r");
+    std::string rfileContents(ss.str());
+    boost::tokenizer<boost::char_separator<char> > tok(rfileContents, sep);
```

Comment:
Do we need to use the Boost tokenizer? This is just splitting the file by ` \n\r`. This should be easily done with standard algorithms. I would prefer not to add another Boost dependency here.

Reply:
We don't need, this was simply done because of the example int he `Boost` code, I will chang it.

Reply:
Changed.